test/datasource/metadata.spec.js

```diff
@@ -64,4 +64,16 @@ describe('datasource/terraform', () => {
     addMetaData(dep, datasource, lookupName);
     expect(dep).toMatchSnapshot();
   });
+
+  it('Should handle parsing/converting of GitHub sourceUrls with http and www correctly', () => {
+    const dep = {
+      sourceUrl: 'http://www.github.com/mockk/mockk/',
```

Comment:
https://github.com/mockk/mockk/blob/6ac5804751189e57c98fca3154ceb306a256b73c/gradle/upload-archives.gradle#L29 is the case.